tests/common/devices/sonic.py

```diff
@@ -2068,6 +2069,40 @@ def show_syslog(self):
         """
         return self.show_and_parse('show syslog')
 
+    def get_acl_counter(self, acl_table_name, acl_rule_name,
+                        timeout=ACL_COUNTERS_UPDATE_INTERVAL_IN_SEC * 3,
```

Comment:
ACL_COUNTERS_UPDATE_INTERVAL_IN_SEC * 2 should be enough?

Reply:
Thanks @bingwang-ms, I ran some ACL test on different platforms and I think `*2` is enough. Updated it in my latest commit.